### ISSUE TITLE
Require manifest.json file

### ISSUE THREAD

**Issue by webdiscus** [webdiscus/pug-plugin#71]
### Discussed in https://github.com/webdiscus/pug-plugin/discussions/70

<div type='discussions-op-text'>

<sup>Originally posted by **elldur** March 21, 2023</sup>
```
link(rel="manifest" href=require("/meta/webmanifest.json"))
\/
<link rel="manifest" href="{... json contents}">
```
Despite `{test: /\.(ico|svg|png|ttf|json)$/, type: "asset/resource"}` being in webpack config. Loading works correctly when importing in javascript (using import not require). The problem is fixed by using a different extension e.g. `.webmanifest`.

`require` treats json differently even outside of pug but the solutions I found involve changing `require.extensions` which does not exist in pug since require seems to be a wrapper for `mod.require` which is not accessible within pug.

Is there a way around this other than changing the extension?</div>

**Comment by webdiscus** [comment]
Hello @elldur,

thanks for the issue report.
I can reproduce the problem.

Expected:
```
<link rel="manifest" href="output/path/webmanifest.json">
```

Received:
```
<link rel="manifest" href="{&quot;name&quot;:&quot; ...}">
```

I will try to fix it.

**Comment by webdiscus** [comment]
it will be fixed in next major version 5.0.0

**Comment by webdiscus** [comment]
The issue is fixed in the new pug plugin `v5.0.0` release based on the [html-bundler-webpack-plugin](https://github.com/webdiscus/html-bundler-webpack-plugin)
